src/api/integrations/channel/whatsapp/whatsapp.baileys.service.ts

```
@@ -1240,62 +1183,60 @@ export class BaileysStartupService extends ChannelStartupService {
-              },
+              where: { instanceId: this.instanceId },
+              include: { OpenaiCreds: true },
             });
 
             if (openAiDefaultSettings && openAiDefaultSettings.openaiCredsId && openAiDefaultSettings.speechToText) {
-              messageRaw.message.speechToText = await this.openaiService.speechToText(
-                openAiDefaultSettings.OpenaiCreds,
-                received,
-                this.client.updateMediaMessage,
-              );
+              messageRaw.message.speechToText = `[audio] ${await this.openaiService.speechToText(received, this)}`;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)